app/models/feedback_message.rb

```diff
@@ -23,8 +23,4 @@ def abuse_report?
   def capitalize_status
     self.status = status.capitalize unless status.blank?
   end
-
-  def email_messages
```

Comment:
Am I getting this error because you removed this method?
![](https://cl.ly/7dc4276b4235/download/Image%202018-10-08%20at%203.22.08%20PM.png)

Reply:
Ah yep, good catch 👍